examples/pytorch_simple.py

```diff
@@ -144,16 +137,10 @@ def objective(trial):
                 break
             data, target = data.to(DEVICE), target.to(DEVICE)
             output = model(data)
-            test_loss += F.nll_loss(output, target, reduction='sum').item()  # Sum up batch losses.
             pred = output.argmax(dim=1, keepdim=True)  # Get the index of the max log-probability.
             correct += pred.eq(target.view_as(pred)).sum().item()
 
-    test_loss /= N_TEST_EXAMPLES
     accuracy = 100. * correct / N_TEST_EXAMPLES
```

Comment:
I think `100. *` is redundant.

Reply:
Fixed: https://github.com/pfnet/optuna/pull/394/commits/f27b4f65c769717e7036ea26087ca400ee32067b. Thanks!